core/src/com/unciv/ui/pickerscreens/TechPickerScreen.kt

```diff
@@ -65,12 +65,18 @@ class TechPickerScreen(internal val civInfo: CivilizationInfo, centerOnTech: Tec
         // per default show current/recent technology,
         // and possibly select it to show description,
         // which is very helpful when just discovered and clicking the notification
-        val tech = if (centerOnTech != null) centerOnTech else civInfo.tech.currentTechnology()
+        val tech = centerOnTech ?: civInfo.tech.currentTechnology()
```

Comment:
Please revert this - is more readable for non-native kotlin readers

Reply:
Excuse me, but unfortunately, I have to disagree with you, since "non-native kotlin readers" - that's me :) 

First of all, the "Elvis operator" is quite an obvious useful thing, which is recommended by Google - https://developer.android.com/kotlin/common-patterns#handle-nullability
so I assume that's a good code style.
As for me, reducing the number of operators - improves readability: less code - less to read.

Secondly, I have already seen it here in other places, so it is not something new for this project.

And lastly, chosing Kotlin as a programming language and not using its features - isn't it illogical?

Reply:
Well, if you're sure